### PR TITLE
Update Chinese Localization

### PR DESCRIPTION
**Pull request by nageih**
resolve #165 
I think it's better than #165,because it's been reviewed by others and fixed a few wrong translation
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/3756

### CODE REVIEW

**Comment by Noaaan** [comment]
I agree that its inconvenient that the keys aren't sorted. If you'd like I could push a commit that sorts them out, as I am planning to do so anyway when going through them

**Comment by nageih** [comment]
Because I use weblate to translate, it is possible that the order of the keys changed. If you will sort them, I think it's fine

**Comment by Noaaan** [comment]
I have gone ahead and sorted the translation keys to make them easier to review, as well as added missing translation keys from upstream. Quality-wise I can only assume with so many eyes on it that it is good - thank you for this contribution!
If you could compile a list of the contributors to this PR that would be great, as I wish to add that to the CREDITS.md file.

File: src/main/resources/assets/mythicmetals/lang/zh_cn.json [hunk 1]
```diff
@@ -1,727 +1,1201 @@
...
+	"tooltip.midas_gold.level.12": "如果最初，你没有成功……",
+	"tooltip.midas_gold.level.13": "……试一下，再试一下。这样你就会充满勇气。",
+	"tooltip.midas_gold.level.14": "皇家迈达斯金剑的真正实力只会在那个时候展现……",
+	"tooltip.midas_gold.level.15": "……一旦你到达了旅途的终点。",
+	"tooltip.midas_gold.level.16": "经过千百次折叠锻打，剑刃变得纯净。",
+	"tooltip.midas_gold.level.17": "你击败的任何事物都会给你更多你真正渴望的东西：",
+	"tooltip.midas_gold.level.18": "更多的迈达斯金！",
+	"tooltip.midas_gold.level.19": "最后一点了，你可以的！",
+	"tooltip.midas_gold.level.20": "你的剑中已蕴含了迈达斯国王的无上力量",
+	"tooltip.midas_gold.level.21": "chyz在这里",
```
Comment (Noaaan) [inline src/main/resources/assets/mythicmetals/lang/zh_cn.json]: This is a joke line, and can be replaced with something similar. An example is in the German PR: https://github.com/Noaaan/MythicMetals/pull/187/files#diff-b810846f6a469a7519ba3d121dc5d0bb1fcbd8dfd61d4f8de5e2bd46e1b751f4R731

Comment (nageih) [reply src/main/resources/assets/mythicmetals/lang/zh_cn.json]: ok,I will update missing translation key and this key later

**Comment by nageih** [comment]
I have updated most of texts,but I don't know what `Aquarium Resonator` and `Aquarium Steward` are,so I don't translate it

**Comment by Noaaan** [comment]
These two are new blocks for an upcoming Aquarium Rework. They are blocks that work inside the range of an active Conduit (between 32-96 blocks, depending on how much prismarine you place around it)

The Aquarium Steward is a turret of sorts. It acts as an active conduit in the sense it targets and damages nearby mobs. The benefit of it is its better damage and essentially being able to extend the range of protection around your base (with a Conduit as a heart). The name "Steward" was picked in the sense of the turret taking care of/protecting the area.

The Aquarium Resonator is basically a mini-beacon, which provides Beacon Power 2 around it. A small known fact is that Beacon Power acts as Haste, so this is essentially a small Haste 2 beacon for underwater use. The use of the word here is in the sense of sharing emotions, buffing the player.

**Comment by nageih** [comment]
thanks to the answer,now I've done

**Pull request merged**